model/point.py

```diff
@@ -15,3 +16,62 @@ def __init__(
         self.z = z
         self.intensity = intensity
         self.rgb = rgb
+
+    def __repr__(self):
+        return f"Point(xyz=({self.x}, {self.y}, {self.y}), rgb={self.rgb}, i={self.intensity})"
+
+# TODO: Make features optional
+class PointList:
```

Comment:
Hard for me to read with hardcoded numbers. Maybe use some index mappings constants so the words are self descriptive and we can change them in one place. Having tests would make usage more obvious and test your indexes. Notice that the majority of the code is covered in tests to the students what good code looks like. Would also be good to create comments on the point of this class as it wouldn't be so obvious for the students on how to use it.